packages/core/directives/map.spec.ts

```diff
@@ -17,6 +17,8 @@ class MockGoogleMapsAPIWrapper {
   clearInstanceListeners = jest.fn();
   subscribeToMapEvent = jest.fn().mockReturnValue(new Subject());
   getNativeMap = jest.fn().mockReturnValue(Promise.resolve({}));
+  fitBounds = jest.fn().mockReturnValue(Promise.resolve({}));
+  setMapOptions = jest.fn().mockReturnValue(Promise.resolve({}));
```

Comment:
`GoogleApiWrapper#setMapOptions` _does not_ return a Promise atm, so neither should its mock.

Reply:
alternatively, it's just as good to make GoogleApiWrapper#setMapOptions return a promise. Take whichever path you prefer, just keep it consistent.